lib/fluent/log.rb

```diff
@@ -292,7 +292,11 @@ def event(level, args)
       }
 
       map.each_pair {|k,v|
-        message << " #{k}=#{v.inspect}"
+        if k == "error".freeze
+          message << " error_class=#{v.class.to_s} error=#{v.to_s.inspect}"
```

Comment:
`v` would be a string like https://github.com/fluent/fluentd/blob/88054a0b4299845c6696335876cba03bb33ea6e3/lib/fluent/output.rb#L127, so need to add `and v.is_a?(Exception)`.